### ISSUE TITLE
Bug: QF for "Make closure static" misses space after static keyword

### ISSUE THREAD

**Issue by tlamy** [hammer-tools/php-hammer#14]
A really low hanging fruit, but my Kotlin/Java knowledge seems to be beyond zero ;)

### Before QF:
```
$strings = ['a','b','c','d'];
$sorted = usort(fn(string $str1, string $str2) => $str2 <=> $str1, $strings);
```
### After QF:
```
$strings = ['a','b','c','d'];
$sorted = usort(staticfn(string $str1, string $str2) => $str2 <=> $str1, $strings);
```

**Comment by rentalhost** [comment]
It is a rather strange situation, as it is the Jetbrains API itself that is responsible for adding this space. But I also noticed this bug, usually when I do actions too fast. I'll see what I can do! Thanks!

**Comment by rentalhost** [comment]
**Implemented in version 0.22.0:**

Hello @akar1n, the new **version 0.22.0** has just been released and is available for direct download from the **[Releases](https://github.com/hammer-tools/php-hammer/releases/tag/0.22.0)** page. This update will be available on the **[Jetbrains Market](https://plugins.jetbrains.com/plugin/19515-php-hammer)** in approximately three days if you prefer to wait.

**This version comes with some new features, now totaling 51 inspections:**

- Support for the **new PhpStorm 2023.2 EAP** (_minimum version from now on_);
- Inspection to suggest strict comparison when possible;
- Inspection to suggest the use of non-deprecated functions (e.g., `mt_rand()` to `random_int()`);
- Additional feature: add missing _semicolon_ when saving a PHP file;
- Other minor improvements.

Regarding your report, this issue has also been fixed in this version. Since it is a problem with JetBrains' API itself, I couldn't do anything special. In this case, I simply disabled the inspection in Blade files where the issue occurred. If it is affecting regular PHP files, please let me know and send me a sample file so that I can reproduce it here.
